### ISSUE TITLE
[php-apache:ubuntu-16.04] Syntax error on line 4 of /opt/docker/etc/httpd/conf.d/10-php.conf

### ISSUE THREAD

**Issue by sergeyklay** [webdevops/Dockerfile#137]
```
Creating config file /etc/php/7.0/apache2/php.ini with new version
Module mpm_event disabled.
Enabling module mpm_prefork.
apache2_switch_mpm Switch to prefork
apache2_reload: Your configuration is broken. Not restarting Apache 2
apache2_reload: AH00526: Syntax error on line 4 of /opt/docker/etc/httpd/conf.d/10-php.conf:
apache2_reload: FastCgiExternalServer /usr/lib/cgi-bin/php5-fcgi: invalid value for -host: missing port specification
apache2_invoke: Enable module php7.0
apache2_reload: Your configuration is broken. Not restarting Apache 2
apache2_reload: AH00526: Syntax error on line 4 of /opt/docker/etc/httpd/conf.d/10-php.conf:
apache2_reload: FastCgiExternalServer /usr/lib/cgi-bin/php5-fcgi: invalid value for -host: missing port specification
Setting up php7.0-phalcon (3.0.1-14) ...
```

**Steps to reproduce**

Create `Dockerfile` as follows:
```dockerfile
FROM webdevops/php-apache:ubuntu-16.04

RUN \
    curl -s https://packagecloud.io/install/repositories/phalcon/stable/script.deb.sh | bash \
    && /usr/local/bin/apt-install \
        php7.0-phalcon \
        libapache2-mod-php7.0 \
    && a2enmod php7.0 rewrite
```

Run `docker build -t <image>:<tag> --no-cache --rm .`

**Comment by sergeyklay** [comment]
Also, as you can see

> apache2_reload: FastCgiExternalServer /usr/lib/cgi-bin/php5-fcgi: invalid value for -host: missing port specification

On PHP 7

> /usr/lib/cgi-bin/php5-fcgi

**Comment by mblaschke** [comment]
That's because there are placeholders for apache configuration so we need to change them.. hm

**Comment by sergeyklay** [comment]
Ya, `cat /opt/docker/etc/httpd/conf.d/10-php.conf`:
```apache
AddHandler php5-fcgi .php
Action php5-fcgi /php5-fcgi
Alias /php5-fcgi /usr/lib/cgi-bin/php5-fcgi
FastCgiExternalServer /usr/lib/cgi-bin/php5-fcgi -host 127.0.0.1:9000 -pass-header Authorization -idle-timeout 600

<Directory /usr/lib/cgi-bin>
  <IfVersion < 2.4>
      Allow from all
  </IfVersion>
  <IfVersion >= 2.4>
      Require all granted
  </IfVersion>
</Directory>
```

**Comment by sergeyklay** [comment]
The same as https://github.com/webdevops/Dockerfile/blob/develop/docker/php-apache-dev/ubuntu-16.04/conf/etc/httpd/conf.d/10-php.conf

**Comment by sergeyklay** [comment]
Also I see
```
apache2_reload: AH00526: Syntax error on line 6 of /opt/docker/etc/httpd/conf.d/10-server.conf:
apache2_reload: DocumentRoot must be a directory
```

In my opinion if `WEB_DOCUMENT_ROOT` does not exists at start time it should be create.

```
invoke-rc.d: could not determine current runlevel
invoke-rc.d: policy-rc.d denied execution of restart.
```

This can be fixed by:
```sh
# Avoid ERROR: invoke-rc.d: policy-rc.d denied execution of start
echo "#!/bin/sh\nexit 0" > /usr/sbin/policy-rc.d
```

**Comment by mblaschke** [comment]
Thanks for your contribution.. will try to find a good solution :)

**Comment by mblaschke** [comment]
Oh, you shouldn't install `libapache2-mod-php7.0` because that's the mod_php module and we're using FPM (which is recommended) :)

**Comment by sergeyklay** [comment]
Well, `10-php.conf` should be:
```apache
AddHandler php7-fcgi .php
Action php7-fcgi /php7-fcgi
Alias /php7-fcgi /usr/lib/cgi-bin/php7-fcgi
FastCgiExternalServer /usr/lib/cgi-bin/php7-fcgi -host <PHP_SOCKET> -pass-header Authorization -idle-timeout 600

<Directory /usr/lib/cgi-bin>
  <IfVersion < 2.4>
      Allow from all
  </IfVersion>
  <IfVersion >= 2.4>
      Require all granted
  </IfVersion>
</Directory>
```

**Comment by mblaschke** [comment]
We will remove the version from the php stuff:

```
AddHandler php-fcgi .php
Action php-fcgi /php-fcgi
Alias /php-fcgi /usr/lib/cgi-bin/php-fcgi
FastCgiExternalServer /usr/lib/cgi-bin/php-fcgi -host <PHP_SOCKET> -pass-header Authorization -idle-timeout 600

<Directory /usr/lib/cgi-bin>
  <IfVersion < 2.4>
      Allow from all
  </IfVersion>
  <IfVersion >= 2.4>
      Require all granted
  </IfVersion>
</Directory>
```

But this really doesn't matter because these are just alias names for the configuration

**Comment by sergeyklay** [comment]
Anyway this should be fixed

**Comment by sergeyklay** [comment]
Fixed in `0.57.0`.
Thanks!
